tests/util/models/gpflow/models.py

```diff
@@ -148,7 +148,7 @@ def covariance_between_points(
     def reparam_sampler(
         self: GaussianProcessWithoutNoise, num_samples: int
     ) -> ReparametrizationSampler[GaussianProcessWithoutNoise]:
-        return GaussianProcessSampler(num_samples, self)
+        return IndependentReparametrizationSampler(num_samples, self)
 
```

Comment:
This is good. I was going to ask where this GaussianProcessSampler came from. Not sure it was ever needed

Reply:
It's because the shapes my acquisition functions were expecting were different to the shapes `IndependentReparametrizationSampler` was giving, so I had to write my own utility `GaussianProcessSampler` to give the correct shapes. Probably not the best move in retrospect...